transcom/mymove
# INT | B-18497 B-18630 | closeout office shipment card

## [B-18497](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18497)
## [B-18630](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18630)

## Summary

Requested addition from govt. to #12256.

This adds a closeout field to the shipment card when a user finishes setting up a PPM shipment.

### How to test

Login as a user and create a PPM shipment. Before signing, on the home screen that shows the yellow PPM shipment card, you should now see a new Closeout Location field, that contains the office you chose when making the shipment.
